workflows/phageannotator/tests/run_instrain.nf.test

```diff
@@ -0,0 +1,96 @@
+nextflow_workflow {
+
+    name "Test workflow: PHAGEANNOTATOR"
+    script "workflows/phageannotator/main.nf"
+    workflow "PHAGEANNOTATOR"
+
+    // Dependencies
+    tag "SEQKIT_SEQ"
+    //tag"APPENDSCREENHITS"
```

Comment:
Is it necessary to keep these comments them if the the test doesn't include them anymore? Or do you expect some changes?

Reply:
Not necessary to keep, I will remove them!